python/ml4ir/applications/ranking/model/losses/listwise_losses.py

```diff
@@ -6,23 +6,22 @@
 from ml4ir.applications.ranking.model.losses.loss_base import ListwiseLossBase
 
 
-class RankOneListNet(ListwiseLossBase):
+class SoftmaxCrossEntropy(ListwiseLossBase):
```

Comment:
Are there any references we can add? Not paragraphs but at least a pointer for someone to understand a bit more. For instance, to me the difference with `categorical_cross_entropy` is not evident (I am lacking ranking context). Is is suitable both for binary and more fine-grained relevance?